ui/controls.js

```diff
@@ -1717,6 +1736,101 @@ shaka.ui.Controls = class extends shaka.util.FakeEventTarget {
     this.onMouseLeave_();
   }
 
+  /**
+   * Returns if a VR is supported.
+   *
+   * @return {boolean}
+   * @export
+   */
+  isPlayingVR() {
+    goog.asserts.assert(this.vr_ != null, 'Should have a VR manager!');
+    return this.vr_.isPlayingVR();
+  }
+
+  /**
+   * Reset VR view.
+   */
+  resetVR() {
+    goog.asserts.assert(this.vr_ != null, 'Should have a VR manager!');
+    this.vr_.reset();
+  }
+
+  /**
+   * Get the angle of the north.
+   *
+   * @return {?number}
+   * @export
+   */
+  getVRNorth() {
+    goog.asserts.assert(this.vr_ != null, 'Should have a VR manager!');
+    return this.vr_.getNorth();
+  }
+
+  /**
+   * Returns the field view.
+   *
+   * @return {?number}
+   * @export
+   */
+  getVRFieldView() {
+    goog.asserts.assert(this.vr_ != null, 'Should have a VR manager!');
+    return this.vr_.getFieldView();
+  }
+
+  /**
+   * Set the field view.
+   *
+   * @param {number} fieldView
+   * @export
+   */
+  setVRFieldView(fieldView) {
+    goog.asserts.assert(this.vr_ != null, 'Should have a VR manager!');
+    this.vr_.setFieldView(fieldView);
+  }
+
+  /**
+   * Toggle stereoscopic mode.
```

Comment:
How would the app track the current mode?

Reply:
I will add a method to get if it is in the mode or not